backend/spec/helpers/admin/navigation_helper_spec.rb

```diff
@@ -37,8 +37,7 @@
 
       context "when match_path option is supplied" do
         before do
-          allow(helper).to receive(:admin_path).and_return("/somepath")
```

Comment:
This didn't appear to provide any value to me and was causing issues in the test. If somebody can tell me why this is important to have I'll figure out a way to add it back in.